js/event-utils.js

```diff
@@ -0,0 +1,25 @@
+export function getFilteredEvents(events, filters) {
```

Comment:
Note: this is not filtering by zipcode cause the map doesn't really filter by zipcode, it just zooms in based on a zipcode (at least for now). But the events list _will_ need to really filter it down to those within 50 miles of the zipcode, so we don't have a ginormous list to render.